tests/models/test_anthropic.py

```
@@ -10443,21 +10445,115 @@ async def test_anthropic_count_tokens_error(allow_model_requests: None, anthropi
     assert exc_info.value.model_name == model_id
 
 
-async def test_anthropic_bedrock_count_tokens_not_supported(env: TestEnv):
-    """Test that AsyncAnthropicBedrock raises UserError for count_tokens."""
+async def test_anthropic_bedrock_count_tokens_unexpected_response(env: TestEnv):
     from anthropic import AsyncAnthropicBedrock
 
     bedrock_client = AsyncAnthropicBedrock(
         aws_access_key='test-access-key',
         aws_secret_key='test-secret-key',
```

Ruff (0.15.18)

[error] Possible hardcoded password assigned to argument: "aws_secret_key"

(S106)